### PR TITLE
Changing AVM codec description

### PR DESCRIPTION
**Pull request by Jamaika1**

### CODE REVIEW

File: apps/avifenc.c [hunk 1]
```diff
@@ -235,7 +235,7 @@ static void syntaxLong(void)
     printf("    -g,--grid MxN                     : Encode a single-image grid AVIF with M cols & N rows. Either supply MxN identical W/H/D images, or a single\n");
     printf("                                        image that can be evenly split into the MxN grid and follow AVIF grid image restrictions. The grid will adopt\n");
     printf("                                        the color profile of the first image supplied.\n");
-    printf("    -c,--codec C                      : AV1 codec to use (choose from versions list below)\n");
+    printf("    -c,--codec C                      : AV1/AV2 codec to use (choose from versions list below)\n");
```
Comment (y-guyon) [inline apps/avifenc.c:238]: `AVIF_CODEC_AVM` is an experimental feature, so I would avoid mentioning AV2 if `AVIF_CODEC_AVM` is OFF.

Suggestion:
```c
printf(" -c,--codec C       : codec to use (choose from versions list below)\n");
```

File: apps/avifenc.c [hunk 2]
```diff
@@ -790,7 +790,7 @@ static avifBool avifGetBestCellSize(const char * dimensionStr, uint32_t numPixel
 
     // The maximum AV1 frame size is 65536 pixels inclusive.
     if (*cellSize > 65536) {
-        fprintf(stderr, "ERROR: Cell size %u is bigger %s than the maximum AV1 frame size 65536.\n", *cellSize, dimensionStr);
+        fprintf(stderr, "ERROR: Cell size %u is bigger %s than the maximum AV1/AV2 frame size 65536.\n", *cellSize, dimensionStr);
```
Comment (y-guyon) [inline apps/avifenc.c:793]: Suggestion:

```c
fprintf(stderr, "ERROR: Cell size %u is bigger %s than the maximum codec frame size 65536.\n", *cellSize, dimensionStr);
```

File: apps/avifenc.c [hunk 3]
```diff
@@ -1207,7 +1213,6 @@ static avifBool avifEncodeImagesFixedQuality(const avifSettings * settings,
            gainMapStr,
            encoder->autoTiling ? "automatic tiling" : manualTilingStr,
            settings->jobs);
```
Comment (y-guyon) [inline apps/avifenc.c:1215]: Suggestion:

```c
    printf("Encoding with codec '%s' speed [%s], color quality [%d (%s)], alpha quality [%d (%s)]%s, %s, %d worker thread(s), please wait...\n",
           codecName ? codecName : "none",
           speedStr,
           encoder->quality,
           qualityString(encoder->quality),
           encoder->qualityAlpha,
           qualityString(encoder->qualityAlpha),
           gainMapStr,
           encoder->autoTiling ? "automatic tiling" : manualTilingStr,
           settings->jobs);
```

File: apps/avifenc.c [hunk 4]
```diff
@@ -1827,7 +1832,7 @@ MAIN()
             } else {
                 const char * codecName = avifCodecName(settings.codecChoice, AVIF_CODEC_FLAG_CAN_ENCODE);
                 if (codecName == NULL) {
-                    fprintf(stderr, "ERROR: AV1 Codec cannot encode: %s\n", arg);
+                    fprintf(stderr, "ERROR: AV1/AV2 Codec cannot encode: %s\n", arg);
```
Comment (y-guyon) [inline apps/avifenc.c:1835]: Suggestion:

```c
fprintf(stderr, "ERROR: Codec %s is unrecognized, unavailable or can only decode\n", arg);
```

File: apps/avifenc.c [hunk 5]
```diff
@@ -2513,10 +2518,19 @@ MAIN()
...
+            printf(" * Gain Map AV1 total size: %" AVIF_FMT_ZU " bytes\n", byteSizes.gainMapSizeBytes);
+        }
+    }
+    if (settings.codecChoice == AVIF_CODEC_CHOICE_AVM) {
+        printf(" * Color AV2 total size: %" AVIF_FMT_ZU " bytes\n", byteSizes.colorSizeBytes);
+        printf(" * Alpha AV2 total size: %" AVIF_FMT_ZU " bytes\n", byteSizes.alphaSizeBytes);
+        if (byteSizes.gainMapSizeBytes > 0) {
+            printf(" * Gain Map AV2 total size: %" AVIF_FMT_ZU " bytes\n", byteSizes.gainMapSizeBytes);
+        }
     }
```
Comment (y-guyon) [inline apps/avifenc.c:2534]: Suggestion:

```c
    printf(" * Coded color total size: %" AVIF_FMT_ZU " bytes\n", byteSizes.colorSizeBytes);
    printf(" * Coded alpha total size: %" AVIF_FMT_ZU " bytes\n", byteSizes.alphaSizeBytes);
    if (byteSizes.gainMapSizeBytes > 0) {
        printf(" * Coded gain map total size: %" AVIF_FMT_ZU " bytes\n", byteSizes.gainMapSizeBytes);
    }
```

**Review by y-guyon — CHANGES_REQUESTED** [review]
Thank you for your contribution.

These changes should also be made in `avifdec`.

**Comment by vrabaud** [comment]
Hi, can you please rebase and take the comments into account. Thx

**Review by y-guyon — APPROVED** [review]

**Comment by y-guyon** [comment]
Thank you for rebasing. There was no notification so we forgot to check this pull request afterwards.

I rebased it myself again.

**Pull request merged**